apps/web/src/pages/studies/index.tsx

```diff
@@ -56,8 +54,19 @@ export default function Studies({
 }: StudiesProps) {
   const router = useRouter()
 
+  const [dashboardLink, setDashboardLink] = useState('')
+  const [isClient, setIsClient] = useState(false)
```

Comment:
It looks like `isClient` is always true (eventually). Is this laying the foundation for something else?